.github/workflows/cncf-conformance.yaml

```diff
@@ -94,6 +94,40 @@ jobs:
             sudo podman exec "${node}" systemctl disable firewalld || true
           done
 
+      - name: Configure hostname resolution for cluster nodes
+        shell: bash
+        run: |
+          set -euo pipefail
+
+          # Add cluster node hostnames to /etc/hosts to enable hostname resolution
+          # from the host where tests run. This is needed because Sonobuoy e2e tests
+          # access kubelet APIs using node names (microshift-okd-1, microshift-okd-2)
+          # which are only resolvable within the podman network by default.
+          echo "Adding cluster node hostnames to /etc/hosts..."
+          for node in $(sudo podman ps --filter name=microshift-okd- --format '{{.Names}}'); do
+            ip=$(sudo podman inspect "$node" | jq -r '.[].NetworkSettings.Networks[].IPAddress')
+            if [ -n "$ip" ] && [ "$ip" != "null" ]; then
+              echo "$ip $node" | sudo tee -a /etc/hosts
+              echo "  ✓ Added: $ip $node"
+            else
+              echo "  ✗ Warning: Could not get IP for $node"
+            fi
+          done
+
+          echo ""
+          echo "Current /etc/hosts entries for microshift-okd nodes:"
+          grep microshift-okd /etc/hosts || true
+
+          echo ""
+          echo "Verifying hostname resolution:"
+          for node in microshift-okd-1 microshift-okd-2; do
+            if host "$node" > /dev/null 2>&1; then
+              echo "  ✓ $node resolves successfully"
+            else
+              echo "  ✗ $node failed to resolve"
+            fi
+          done
+
```

Comment:
I'm not sure this verification is necessary. I think CNCF will do it for us, so we could as well remove this code.

Reply:
May be i would keep this code so that we can get the results faster instead of waiting for  2 hour long and find out that test failed because of this issue ? WDYT ? I can add `exit 1` so that test comes out if hostname resolution fails, WDYT ?